chess/__init__.py

```diff
@@ -626,14 +626,15 @@ def __init__(self, from_square, to_square, promotion=None, drop=None):
         self.promotion = promotion
         self.drop = drop
 
-    def uci(self):
+    def pacn(self):
         """
-        Gets an UCI string for the move.
+        Gets an PACN string for the move.
```

Comment:
It is kind of odd calling it from_uci and from_xboard. I abstracted the implementation out so we could mark it deprecated or something while phasing it out(if we are actually going to remove it)